plugins/vpp/ifplugin/vppcalls/dump_interface_vppcalls.go

```diff
@@ -46,6 +47,17 @@ type InterfaceMeta struct {
 	SwIfIndex    uint32 `json:"sw_if_index"`
 	Tag          string `json:"tag"`
 	InternalName string `json:"internal_name"`
+	Dhcp         *Dhcp  `json:"dhcp"`
+}
+
+// Dhcp is helper struct for DHCP metadata for interfaces using it
+type Dhcp struct {
```

Comment:
Shouldn't this store data from `Lease` from `DhcpClientDetails` as well? It's being dumped already.

Reply:
It will be, as I mentioned